### PR TITLE
Reworked config check scripts to work with pytest

### PR DESCRIPTION
**Pull request by chartjes**
r? @karlht

### CODE REVIEW

File: manifest.ini [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 
 [prod]
-server = https://firefox.settings.services.mozilla.com/v1/ 
+server = https://firefox.settings.services.mozilla.com/v1/
+we_server_url = https://webextensions.settings.services.mozilla.com/v1
+account_server_url = https://api.accounts.firefox.com/v1
+oauth_server_url = https://oauth.accounts.firefox.com/v1
+fxa_user_email =
+fxa_user_password =
+qa_collection = 3e0f013d2362aa4fa7b0db2fefb40011b9545d78dcca7aabc0d32f0b42f1733d
```
Comment (karlht) [inline manifest.ini]: Do we want the `qa_collection` uuid to be the same between stage and prod, or do we care?

File: manifest.ini [hunk 2]
```diff
@@ -1,6 +1,18 @@
 [stage]
 server = https://kinto.stage.mozaws.net/v1
 writer_server = https://kinto-writer.stage.mozaws.net/v1
```
Comment (karlht) [inline manifest.ini]: Does prod have a `writer_server`?

**Comment by karlht** [comment]
Chris, please feel free to merge this whenever you like.  r+

File: manifest.ini (hunk 2 above)
Comment (chartjes) [inline manifest.ini]: prod does have one but QA does not have access to it --- the staging value is used for our stage-deployment testing

File: manifest.ini (hunk 1 above)
Comment (chartjes) [inline manifest.ini]: We don't care because we just need something unique that Firefox will not attempt to read and use

**Pull request merged**